### PR TITLE
First try to add Buenos Aires (Argentina) support

### PR DESCRIPTION
**Pull request by ignaciocaamanio**
I'm sure that I made mistakes, but I do this pull request to get some advice.

1) I have imported the Gradle project in Eclipse and I think that's the reason for 6 files modified.
2) I don't understand the test file, I copied the OntarioProviderLiveTest file and replace "OntarioProvider" with "BsAsProvider". 

Thanks.

### CODE REVIEW

**Comment by schildbach** [comment]
Thanks for your PR! Roughly it looks good. How good does Navitia work for Buenos Aires? Do you have local experience?

The LiveTest is basically there for your convenience: You can add test cases such that all the possibilities of the API is tested.

Yes the project files should not go into the pull request. Usually when commiting a set of changes, you should go through them and unselect any random stuff that was included unintentionally. You can fix it afterwards as well and force-push your branch. But don't worry if you're not up to it, I can fix it as well when merging.

**Comment by ignaciocaamanio** [comment]
Thanks for your reply, it's my first contribution in github and I'm not used Java and Eclipse, I have to learn a lot...

I live in Buenos Aires but didn't test Navitia. I'm going to research how to use it because I don't know how to.

About the project files: I have to avoid them when I commit in Git with `git add [files to add]` and not `git add .` or I have to do something in Eclipse?

Thanks

**Comment by grote** [comment]
Thanks a lot for your PR, @ignaciocaamanio!

Just a small comment on the name of the new provider: The region in navitia is called "ar" for all of Argentina and there might be more cities added later. The name you have chosen is `BsAsProvider`. Maybe something else like `ArProvider` would be better?

> I have to avoid them when I commit in Git with git add [files to add] and not git add . or I have to do something in Eclipse?

If you use git on the command line, you don't need to do anything in Eclipse. Just add only the files that you want to change.

**Comment by ignaciocaamanio** [comment]
Hi @grote ! 
I agree with the change to "ar", but I'm not sure if Navitia provides information about all the country or just Buenos Aires (the capital). Do you know how to check that? I'm trying with the Navitia API but I only see subways from the capital.

**Comment by grote** [comment]
At the moment, they have only the capital, but they might add other regions/cities later like they did for Brazil. You can check what is included at the moment here: https://navitia.opendatasoft.com/explore/dataset/ar/table/

File: enabler/test/de/schildbach/pte/live/ArProviderLiveTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	public ArProviderLiveTest() 
+	{
+		super(new ArProvider(secretProperty("navitia.authorization")));
+	}
+
+	@Test
+	public void nearbyStationsAddress() throws Exception 
+	{
+		nearbyStationsAddress(-34618072, -58436443);
+		System.out.println("\n");
```
Comment (schildbach) [inline enabler/test/de/schildbach/pte/live/ArProviderLiveTest.java]: Can you remove this line? No printing to sysout please.

**Comment by schildbach** [comment]
Generally looks good now. Would you like to squash all your 7 commits together? Maybe you want to adapt the copyright year to 2016?

**Comment by ignaciocaamanio** [comment]
I have changed the copyright year. Also I searched how to "squash" the commits but I don't know if is ok...

**Comment by ignaciocaamanio** [comment]
Do I have to make a new PR ?

**Comment by grote** [comment]
No, it would be enough to rebase on latest master and then force push into your branch.

**Comment by ignaciocaamanio** [comment]
How do I rebase on master? 
I tried `git rebase origin/master ` but says "Current branch master is up to date."

**Comment by grote** [comment]
You might need to run `git fetch origin` first.

**Comment by ignaciocaamanio** [comment]
Ok, I don't know how to do this. I need to learn more Git. Sorry for your time.

**Comment by schildbach** [comment]
Don't worry about rebasing, On this PR I can do that while merging.

Currently there are only 2 tests. Has querying of departures and trips ever been tested? Could you perhaps add a tests for those? I'm asking because a provider that can only query for nearby locations and suggest locations by name isn't really helpful.

**Comment by ignaciocaamanio** [comment]
I've been testing the following code to see what information give me, varying the stop_area:

```@Test
	public void queryDeparturesStopArea() throws Exception
	{
		queryDeparturesStopArea("stop_area:OBS:SA:1059");
	}
```

I take the stop_area data from here: [https://api.navitia.io/v1/coverage/ar/stop_areas]

Some stop areas pass the test and give me correct data, but only of metro. I didn't found train or bus data (?). Some of the stop areas fails the test. 
I will continue testing Navitia to try to understand what is the coverage, but I think is not to good. I'm sorry but is hard to me to understand how Navitia and public-transport-enabler works. Maybe you have some kind of guide for the test functions?

**Comment by grote** [comment]
I think Navitia only has the subways in Buenos Aires, so it would be expected that non-subway stops don't have departures. You cold also check the raw GTFS data from here to be sure: https://navitia.opendatasoft.com/explore/dataset/ar/table/?sort=type_file

As for more examples for other tests, check out Paris: https://github.com/schildbach/public-transport-enabler/blob/master/enabler/test/de/schildbach/pte/live/ParisProviderLiveTest.java

**Comment by ignaciocaamanio** [comment]
Yes, only has subway, what is most used here are the buses. Is not very useful for Buenos Aires...

**Pull request closed without merging**